spec/student_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../student'
+
+describe Student do
+  context 'with valid input' do
+    before(:each) do
+      @new_student = Student.new(18, 'Robert')
```

Comment:
- [Optional] I think you need to create a new instance of a student with three parameters that's `@new_student = Student.new(18, 'Robert', parents_permission)`